README.md

```diff
@@ -1,22 +1,140 @@
 # Pimcore Elasticsearch Plugin
 
-## What it does
+The Elasticsearch plugin for Pimcore Saves/updates contents (text only) of document's (editables) and assets to Elasticsearch. It also provides a query builder to search the index.
 
-* Saves/updates contents (text only) of Document's editables to Elasticsearch
- 
-## Configure
+## Features
 
-Copy the distribution config file (`elasticsearchplugin.xml.dist`) in the root of the plugin folder 
+* Automatic indexing of documents and assets to elasticsearch when created/updated in Pimcore admin. 
+* Hooks to add custom properties to the index.
+* A simple query builder to retrieve indexed documents. Supports querying, filtering, sorting and pagination.
+
+## Installation via composer
+
+The recommended method to install the Pimcore Elasticsearch PLuing is via Composer.
+
+1 - Add byng/pimcore-elasticsearch-plugin as a dependency in your project's composer.json file (change version to suit your version of Elasticsearch):
+```json
+    {
+        "require": {
+            "byng/pimcore-elasticsearch-plugin": "~3.0"
+        }
+    }
+```
+
+2 - Download and install Composer:
```

Comment:
It's not relevant to us but to others it will be, it makes the documentation more complete. Also not everyone is familiar with composer yet unfortunately. This is the standard format which a lot of php projects on github use.